rids/iocs/iocs.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2022 Jigsaw Operations LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Combined IOC feed parser interface for the different formats of IOCs."""
+
+
+import urllib.request
+
+from rids.iocs import bad_ip_list
+from rids import rules
+
+
+_PARSERS = {
+  "BAD_IP_LIST": bad_ip_list.FeedParser,
+}
+
+
+def fetch_iocs(rids_config):
+  """Parses the IOC indicated by `ioc_config` into a RuleSet instance.
+  
+  Args:
+    rids_config: { 'iocs': list[dict{'name', 'url', 'format'}] }
```

Comment:
This is not a list of IOCs, which confused me. They are more like configs or ioc feeds/sources. Perhaps call them feeds/sources?

Reply:
I've renamed it here and in the config to be 'ioc_sources'... I almost went with 'feeds' but I feel like that doesn't really fit for locally-defined or inline-defined IOCs.